app/templates/config.html

```diff
@@ -5,38 +5,42 @@
-            <option name="Cocoa" value="Cocoa">Cocoa</option>
-            <option name="Natural Gas" value="Natural Gas">Natural Gas</option>
-            <option name="Canola" value="Canola">Canola</option>
-            <option name="Cotton" value="Cotton">Cotton</option>
-            <option name="Oat" value="Oat">Oat</option>
-            <option name="Rice" value="Rice">Rice</option>
-            <option name="Soybeans" value="Soybeans">Soybeans</option>
-            <option name="Sugar" value="Sugar">Sugar</option>
-            <option name="Wheat" value="Wheat">Wheat</option>
-            <option name="Beef" value="Beef">Beef</option>
-        </select>
-        <br>
-        <input class="btn btn-outline-info" href='/data-summary/' type="submit" value="Save configuration">
-    </form>
-
-    <!--<a class="btn btn-outline-info" href='/data-summary/' role='button'>Save configuration</a>-->
-</body>
+    <head>
+        <link rel="stylesheet" href="../static/css/style.css">
+        <title>Select</title>
+    </head>
+    
+    <body>
+        <header id="Configuration">
+            <!--<h1>Configuration page</h1>-->
+            <p1> Select items you would like to Track affected by inflation </p1>
+            <p2> *Hold down the Ctrl (windows) / 
+            Command (Mac) button to select multiple options* </p2>
+        </header>
 
+        <form class="watchlistSelection" action="data-summary" method="POST">  
+            <select class="watchlistForm" type="text" name="itemDropdown" placeholder="Select items to watch" multiple>
+                <!--<option disabled selected>Select items</option>-->
+                <option name="Corn" value="Corn">Corn</option>
+                <option name="Arabica Coffee" value="Arabica Coffee">Arabica Coffee</option>
+                <option name="Brent Crude Oil" value="Brent Crude Oil">Brent Crude Oil</option>
+                <option name="Cocoa" value="Cocoa">Cocoa</option>
+                <option name="Natural Gas" value="Natural Gas">Natural Gas</option>
+                <option name="Canola" value="Canola">Canola</option>
+                <option name="Cotton" value="Cotton">Cotton</option>
+                <option name="Oat" value="Oat">Oat</option>
+                <option name="Rice" value="Rice">Rice</option>
+                <option name="Soybeans" value="Soybeans">Soybeans</option>
+                <option name="Sugar" value="Sugar">Sugar</option>
+                <option name="Wheat" value="Wheat">Wheat</option>
+                <option name="Beef" value="Beef">Beef</option>
+            </select> 
+            <br>
+            <!--<input class="btn btn-outline-info" href='/data-summary/' type="submit" value="Save configuration">-->
+         </form>
+        
+        <div class="config_buttom">
+        <a class="btn btn-outline-info" href='/data-summary/' role='button'> Save and Track </a>
```

Comment:
Not sure why this is causing a problem, but doing _href='/data-summary'_ should resolve the routing issue

Reply:
fixed